pages/linux/newusers.md

```diff
@@ -0,0 +1,20 @@
+# newusers
+
+> Update and create new users in batch mode.
+> More information: <https://manned.org/useradd>.
+
+- It reads from a file with the following format:
+
+`<username>:<passwd>:<uid>:<gid>:<user_info>:<home_dir>:<default_shell>`
```

Comment:
Hmm.  This format reference is totally useful, but it's not a command.  In tldr pages we only give some demonstrations on how to run a command, with the various arguments it accepts.

Reply:
I understand your concern but without giving this information, it doesn't make sense to add it to tldr.